build/commands/lib/syncUtils.js

```diff
@@ -44,6 +44,7 @@ function buildDefaultGClientConfig(
         'src/third_party/chromium-variations': null,
       },
       custom_vars: {
+        'checkout_clang_coverage_tools': true,
```

Comment:
downloads llvm-cov, llvm-profdata totaling around ~10mb on macos. 
Alternatively one could call 
`tools/update.py --package=coverage-tools`

but may lead to a revision mismatch and gclient issues on subsequent syncs

Reply:
I agree. Really hard to know what is going with all the logic config.js and syncUtils.js

I'm even considering if we should embrace gclient and use that to fully manage our brave-core checkout.

Reply:
> I'm even considering if we should embrace gclient and use that to fully manage our brave-core checkout.
The problem is that gclient requires a lot of obscure options to properly deal with patches, etc... when updating chromium branches

Reply:
in any case I might look at this because it messes up my gclient file all the time